litinit.go

```diff
@@ -130,5 +133,32 @@ func litSetup(conf *config) *[32]byte {
 		log.Fatal(err)
 	}
 
+	// do UPnP port forwarding
+	// right now we fatal if we aren't able to port forward via upnp
+	// a question though is whether we should continue connceting without
+	// port forwardign
+	if conf.UPnP {
+		// Connect to router
+		var externalIPs []string
+		d, err := UpnP.DiscoverCtx(context.Background())
+		if err != nil {
+			fmt.Printf("Unable to discover router %v\n", err)
+			log.Fatal(err)
+		}
+		// Get external IP
+		ip, err := d.ExternalIP()
+		if err != nil {
+			fmt.Printf("Unable to get external ip %v\n", err)
+			log.Fatal(err)
+		}
+		log.Printf("Your external IP is %s", ip)
+		// Forward peer port
+		err = d.Forward(uint16(conf.Rpcport), "lnd peer port")
```

Comment:
We shouldn't NAT the RPC Port, but the lndc port (2448). That's not configurable rn, because you pass it to `listen` (or configure the automatic Listen port that is used to trigger `listen` on startup). 

It would make sense to do the NAT punch when `listen` is executed, because we know the port we need to punch at that time only.

I noticed when i tried running this, and found the uPnP was opening port 8001 on my router.

This does make me feel this wasn't tested for actual functioning.